stable/oauth2-proxy/templates/google-secret.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and .Values.config.google (not .Values.config.google.existingSecret) }}
+{{- if and .Values.config.google .Values.config.google.serviceAccountJson (not .Values.config.google.existingSecret) }}
```

Comment:
When testing this I did not manage to get `service-account.json:` to work.

Here is my test values.yaml:
```
config:
  google:
    adminEmail: admin@example.org
    serviceAccountJson: Zm9v
```

And test command: `helm template --name test . --values test.yaml`